cdap-app-fabric/src/main/java/co/cask/cdap/metadata/LineageHandler.java

```diff
@@ -80,13 +80,15 @@ public void datasetLineage(HttpRequest request, HttpResponder responder,
                              @QueryParam("start") String startStr,
                              @QueryParam("end") String endStr,
                              @QueryParam("levels") @DefaultValue("10") int levels,
-                             @QueryParam("collapse") List<String> collapse) throws Exception {
+                             @QueryParam("collapse") List<String> collapse,
+                             @QueryParam("rollup") @DefaultValue("none") String rollup) throws Exception {
```

Comment:
I think it is okay to leave the default value as null and make the rollup parameter as nullable in LineageAdmin methods.